### DISCUSSION TITLE
support for azure API endpoints

### DISCUSSION THREAD

**Discussion by raidoh** [danielmiessler/Fabric#149] [Ideas]
would be a nice addition to fabric

**Comment by wally-kroeker** [comment]
Yes please

**Comment by wally-kroeker** [comment]
I am trying to find the part of the code that points at the open AI api.  But I am not a programmer.

**Comment by digitalw00t** [comment]
I think the azure end points for openai can be specified  with an environment variable in your shell.

Example:
export OPENAI_BASE_URL=<end point>
